rcl/src/rcl/context.c

```diff
@@ -28,7 +28,7 @@ extern "C"
 rcl_context_t
 rcl_get_zero_initialized_context(void)
 {
-  static rcl_context_t context = {
+  rcl_context_t context = {
```

Comment:
use stack memory instead, so that we can avoid the racy condition with multi-threaded manner.